src/main/java/org/jabref/logic/cleanup/CleanupWorker.java

```diff
@@ -44,8 +52,18 @@ public List<FieldChange> cleanup(CleanupPreferences preset, BibEntry entry) {
     private List<CleanupJob> determineCleanupActions(CleanupPreferences preset) {
         List<CleanupJob> jobs = new ArrayList<>();
 
+        // Special handling for MSC code conversion
```

Comment:
Use one term for the thing done: "description conversion" maybe? - the `Boolean` variable of the method contains `description` (which is good); this should be done trhoughout the code and ocmments.